sw/device/lib/crypto/include/datatypes.h

```diff
@@ -27,7 +29,7 @@ typedef enum crypto_status {
   // Status is OK; no errors.
   kCryptoStatusOK = 0x4d39,
   // Invalid input arguments; wrong length or invalid type.
-  kCryptoIncorrectInput = 0xbd57,
+  kCryptoStatusInvalidInput = 0xbd57,
```

Comment:
nit: To keep consistency with status codes, I think this should be `kCryptoStatusBadArgs`